examples/dns/package.json

```diff
@@ -30,7 +30,7 @@
     "@opentelemetry/exporter-jaeger": "^0.14.0",
     "@opentelemetry/exporter-zipkin": "^0.14.0",
     "@opentelemetry/node": "^0.14.0",
-    "@opentelemetry/plugin-dns": "^0.12.1",
+    "@opentelemetry/plugin-dns": "^0.13.0",
```

Comment:
The 0.13.0 version of plugin requires 0.15.0 of Otel.

Besides that the examples needs some more work to get them working because of the breaking changes between 0.14.0 and 0.15.0 (and maybe other oversights).

I assume you would like to release 0.13.0 plugins asap. Maybe keep them unmodified (0.12.1/0.14.0) but working for now and fix them as followup once 0.13.0 is availble at npm.

Reply:
Yeah that sounds like a reasonable course of action.